website/docs/developer/research/research-deep-glacier-migration.mdx

```diff
@@ -0,0 +1,242 @@
+
+### Use Cases
+
+Based on the discussion with ORCA team, we came up with two use cases that the end user might be interested in:
+- Convert all the holdings or holdings with a known key prefix from Flexible Retrieval(formerly Glacier) to Deep Archive.
+- Convert specific collection or granules from Flexible Retrieval(formerly Glacier) to Deep Archive.
+
+The details of each use cases and relevant architecture drawings can be found in the next sections.
+
+### Migrating all ORCA holdings to deep archive.
+
+The migrate all holdings use case assumes that the end user wants to migrate all ORCA holdings to the DEEP ARCHIVE storage type or specific collections of data that can be grouped together via a key path like `/ShortName/Version` within the ORCA bucket. The most efficient and cost effective approach to this use case would to have the operator perform the migration steps manually as outlined below.
+
+The steps for performing the migration are as follows:
+1. Configure ORCA default storage type or the collection level storage type overide to DEEP ARCHIVE and re-deploy ORCA if needed. See this [documentation](https://github.com/nasa/cumulus-orca/blob/develop/CHANGELOG.md#migration-notes) for additional details.
+2. Run the lifecycle configuration with transition_days = 0 as shown in the terraform code example below.
+:::note
+The following S3 lifecycle rule will move all objects with under prefix `tmp/` to Deep Archive. For more information on S3 lifecycle policy, see this terraform [documentation](https://registry.terraform.io/providers/PixarV/ritt/latest/docs/resources/s3_bucket_lifecycle_configuration).
+
+```teraform
+resource "aws_s3_bucket_lifecycle_configuration" "bucket-config" {
+  bucket = aws_s3_bucket.bucket.bucket
+
+  rule {
+    id = "transition to deep archive"
+    status = "Enabled"
+
+    transition {
+      days          = 0
+      storage_class = "DEEP_ARCHIVE"
+    }
+    filter {
+      prefix = "tmp/"
+    }
+  }
+
+```
+
+:::tip
+To migrate to a different storage class if needed, change `storage_class` under transition block to the desired class in the terraform code above.
+:::
+
+3. Wait 48 hours for all objects to migrate to deep archive.
+4. Update the storage type in the ORCA Catalog to the value of DEEP ARCHIVE. Below are three approaches that can accomplish this.
+- (Recommended) Have a lifecycle rule applied to the glacier bucket that will trigger a lambda function for every `s3:LifecycleTransition` event in the bucket. This lambda will automatically update the ORCA catalog status table for storage class to DEEP ARCHIVE at the object level.
+- Manually run a temporary lambda function that updates the catalog status table for storage class to DEEP ARCHIVE after a set amount of time (48+ hours).
+- Manually update the table column in the database using SQL.
+5. Validate internal reconciliation for mismatches in storage class.
+6. If no mismatch, disable/delete the lifecycle rules.
```

Comment:
If step 2 modifies the TF files, those should also be reverted.

Reply:
updated to 

`6. If no mismatch, disable and revert back the changes to the lifecycle rules.`